tests/sentry/incidents/test_tasks.py

```diff
@@ -208,7 +208,7 @@ def test_missing_project(self):
         )
 
     def test(self):
-        with patch.object(AlertRuleTriggerAction, "_type_registrations", new={}):
+        with patch.object(AlertRuleTriggerAction, "_factory_registrations", new={}):
```

Comment:
I'm not thrilled about the way these tests require visibility into the private registry. I might take a pass at it while I'm here.